### PR TITLE
Fix parsing of keyword arguments

### LINKED ISSUE #85: If supported_args_permutations is empty test is skipped
If the function `get_supported_args_permutations` returns an empty dict, the tests are essentially skipped.

This is a left over issue of #79

### PR DESCRIPTION
**Pull request by daniel-thom**
All tests pass with PSY commit ID 5b49e6cd2. The latest version of the psy4 branch requires changes here.

Fixes #85

### CODE REVIEW

File: src/system_descriptor_data.jl [hunk 1]
```diff
@@ -1411,7 +1411,6 @@ const SYSTEM_CATALOG = [
         supported_arguments = [
             SystemArgument(;
                 name = :avr_type,
-                default = "AC1A",
```
Comment (daniel-thom) [inline src/system_descriptor_data.jl:1414]: @jd-lara Can you confirm that none of these descriptors should have default values?

Comment (rodrigomha) [reply src/system_descriptor_data.jl:1414]: In my opinion, those descriptors should not have default values. But I did not make that descriptor data, so I'm not sure if there was a reason of having default values there.

**Review by rodrigomha — APPROVED** [review]

File: src/system_descriptor_data.jl (hunk 1 above)
Comment (daniel-thom) [reply src/system_descriptor_data.jl:1414]: The main branch does not have defaults.

Comment (jd-lara) [reply src/system_descriptor_data.jl:1414]: Agree with Rodrigo

**Review by jd-lara — APPROVED** [review]

**Pull request merged**